client/src/pages/CreateSurvey/index.jsx

```diff
@@ -92,15 +92,14 @@ const CreateSurvey = ({ history }) => {
         ...surveyForSending,
         status: 'draft',
         responses: [],
-        dateToClose: null,
         dateClosed: null,
         dateEdited: Date.now(),
         datePublished: null,
-        dateToPublish: null,
         recipients: recipientsToSend,
         recipientIds: recipientsToSend,
       };
       delete surveyForSending.employeeData;
+      delete surveyForSending.modalStyle;
       delete surveyForSending.openCreateSurveyModal;
       delete surveyForSending.isConfirming;
       delete surveyForSending.openModal;
```

Comment:
Preferably this spreading operator should not be used. Need to think of a better way ;D